daemon/logger/journald/read.go

```diff
@@ -392,6 +396,76 @@ func (s *journald) readLogs(logWatcher *logger.LogWatcher, config logger.ReadCon
 	return
 }
 
+func (s *journald) readRemoteLogs(logWatcher *logger.LogWatcher, config logger.ReadConfig) {
+	client := http.Client{}
+
+	// Build a url to get logs from systemd-journal-gatewayd.
+	container_id := s.vars["CONTAINER_ID_FULL"]
+	v := url.Values{}
+	v.Set("CONTAINER_ID_FULL", container_id)
+	if config.Follow {
+		v.Set("follow", "")
+	}
```

Comment:
Supporting `config.Tail` seems possible but `config.Since` is not easy.

Adding `Range: entries=cursor[[:num_skip]:num_entries]` HTTP header is the only way the fetch logs in a specific period. As you see the header requires a cursor string but I don't know how do I get the cursor from a date specified by `since` option.